src/menu/options.c

```diff
@@ -662,9 +662,11 @@ static MenuData* create_options_menu_gamepad(MenuData *parent) {
 
 	OptionBinding *b;
 
+#ifndef __SWITCH__
 	add_menu_entry(m, "Enable Gamepad/Joystick support", do_nothing,
 		b = bind_option(CONFIG_GAMEPAD_ENABLED, bind_common_onoff_get, bind_common_onoff_set)
 	);	bind_onoff(b);
+#endif
 
 	add_menu_entry(m, "Device", do_nothing,
```

Comment:
I'm curious, does this actually work? If I understand right, you can attach multiple controllers to the switch; do you know if they show up here correctly?

Reply:
What happens is that SDL lists 8 controllers no matter what, and then whether you receive inputs or not on any of them will depend of controller actually attached and their order. So what you see in the menus is "Switch controller #1" regardless of the type (pro controller, single joycon, joycon pair, etc)
Btw we could also in the future add an option that would split the joycons, allowing to use a single joycon to play, but i'm not sure if it's worth the trouble tho. (the pair is seen as a single controller by default, and we can talk to the hid service to merge or split any)

Refs:
SDL joystick init: https://github.com/devkitPro/SDL/blob/switch-sdl2/src/joystick/switch/SDL_sysjoystick.c#L74
Hid init: https://github.com/switchbrew/libnx/blob/01d648e7ed211d1e57b6026a1f9fc15cd5c7763a/nx/source/services/hid.c#L106

Reply:
Hm well, that's suboptimal. In that case maybe this option should be hidden (and defaulted to "any device") too, unless it is actually important to actually filter unwanted devices out.

I'm not seeing the benefit of splitting the joycons. It could be useful later if ever add local multiplayer though, but that's probably not happening in the nearest future.